### DISCUSSION TITLE
Why issue was closed?

### DISCUSSION THREAD

**Discussion by DracoClaw** [File-New-Project/EarTrumpet#731] [Q&A] [answered]
So my issue #363 was closed with no reason as to why. Was the feature added to the app?

**Comment by riverar** [comment, marked as answer]
Sorry for the confusion, we're separating issues from feature requests. This particular issue has a link to [the new discussion thread](https://github.com/File-New-Project/EarTrumpet/discussions/684) at the bottom.

Screenshot:
![image](https://user-images.githubusercontent.com/475132/106813777-54a7b700-6626-11eb-873f-c5bc57bfb441.png)

**Comment by DracoClaw** [comment]
oh i see! thanks so much, i guess that part didnt render on mobile

**Reply by riverar** [reply]
Interesting, will pass that feedback on to GitHub!

**Reply by DracoClaw** [reply]
![image](https://user-images.githubusercontent.com/34274162/106946609-fcf67200-6731-11eb-8324-ae878dc005bb.png)

**Reply by riverar** [reply]
Thanks, the mobile client is definitely lacking in many areas.
